package-lock.json

```diff
@@ -19559,9 +19450,9 @@
       }
     },
     "rotating-file-stream": {
-      "version": "3.0.4",
-      "resolved": "https://registry.npmjs.org/rotating-file-stream/-/rotating-file-stream-3.0.4.tgz",
-      "integrity": "sha512-Zd+IA5soqBUKtCa7KfMRnF7GKEaRav/dJFU7UHc+r1FMd8VvMLUFE0q1HX5s6XJO1cZ6jAqtTa/ZFnjc7IqlJA=="
+      "version": "3.0.0",
```

Comment:
Same here. Downgraded `rotating-file-stream` instead of updating it to the latest **v3.1.0**.